src/import-data/ImportDataDestination.ts

```diff
@@ -219,6 +220,12 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
           ]),
       ]),
       importMayTakeTime && div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
+      !!selectedWorkspace &&
+        h(Fragment, [
+          h(WorkspacePolicies, { workspace: selectedWorkspace }),
+          !!isProtectedData &&
+            div({}, [p({}, ['Importing this data may add:']), ul({}, [li({}, ['Additional access controls'])])]),
```

Comment:
The props object can be omitted when it's empty.

```suggestion
            div([p(['Importing this data may add:']), ul([li(['Additional access controls'])])]),
```